src/content/components/tooltip/usage.mdx

```diff
@@ -2,9 +2,35 @@
 title: Tooltip
 tabs: ['Code', 'Usage','Style']
 ---
+
 ## General guidance
 
-Tooltips provide additional information upon hover or focus.
+Tooltips provide additional information upon hover or focus. The information should be contextual, useful, and nonessential information. Keep tooltips short.
+
+## Placement
+
+Icon tooltips and interactive tooltips may be positioned **top**, **below**, **left**, or **right** to the trigger item. The container of the tooltip text may be aligned to **start**, **center** or **end**.
```

Comment:
Super knit-picky but should it say "bottom" instead of "below

Reply:
I actually think that it should be "above" and "below" 😂, but pretty much agree

Reply:
Right, language wise I don't mind changing to any of your options. But to follow the code terminology, I think I will change it to "top" and "bottom" if that makes sense to everyone. 
![image](https://user-images.githubusercontent.com/15144993/58967494-e5573100-8779-11e9-95ed-efdf714d7f40.png)